### DISCUSSION TITLE
Homebrew Dongle Emulator

### DISCUSSION THREAD

**Discussion by mckaleman** [Homebrew/discussions#693] [Everyday usage]
I am new tot he Wii and Homebrew. I have purchased a Wii and a few accessories such as the Rock Band 2 Drum set and Tony Hawk Skateboard. I do have the Dongles but am wondering if there is a software package to use these accessories without always using the dongle.

Thanks!

**Comment by SMillerDev** [comment]
Wrong homebrew, this is the forum for https://brew.sh/ and you're looking for https://wiihomebrew.org/
